portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -128,9 +135,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       String imageUrl = getUploadedFileUrl(request, "image");
 
       String messageRepliesString = getParameter(request, "replies", "");
-	    String messageRepliesArray[] = messageRepliesString.split(",");
-	    List<String> messageReplies = new ArrayList<String>();
-	    messageReplies = Arrays.asList(messageRepliesArray);
+      String messageRepliesArray[] = messageRepliesString.split(",");
```

Comment:
getParameter() could return null if the parameter doesn't exist, which means you would need to have a check non-null before using "messageRepliesString". Otherwise, it will be an exception.